system/settings/settings.xml

```diff
@@ -2192,12 +2192,6 @@
           <control type="spinner" format="string" />
         </setting>
         <setting id="videoscreen.limitedrange" type="boolean" label="36042" help="36359">
-          <requirement>
-            <or>
-              <condition>HAS_GL</condition>
-              <condition>HAS_DX</condition>
-            </or>
-          </requirement>
```

Comment:
ah ok, so RPi, amlogic, etc that don't use LinuxRendererGLES will need to be excluded then.